docs/intro/overview.rst

```diff
@@ -0,0 +1,18 @@
+===============
+Getting started
+===============
```

Comment:
I think this content can be put directly in index.rst, before the links. "Please, see also our :ref:`intro-install` and our :ref:`intro-tutorial` for a quick start." can just refer to the links below. It seems index.rst won't become much larger, but it'd give some information about what's the package about, without making user to visit any additional links.

Reply:
Done.

- moved overview content to index
- removed overview page
- removed links from the index page since we already have toctrees